test/integration/enumerate_databases.test.ts

```diff
@@ -141,6 +142,64 @@ describe('listDatabases()', function () {
     );
   });
 
+  describe('nameOnly option', function () {
+    let client: MongoClient;
+    const DBS = 10;
+    const nameOnlyOptions = [true, false, undefined];
+
+    beforeEach(async function () {
+      client = await this.configuration.newClient({}, { monitorCommands: true }).connect();
+      for (let i = 0; i < DBS; i++) {
+        const db = client.db(`testDb_${i}`);
+        await db.collection('test').insertOne({ a: 1 });
```

Comment:
Are we inserting documents only to create the collection?  

```suggestion
await client.db(<>).createCollection('test');
```